### PR TITLE
fixed link on the doc badge

### PR DESCRIPTION
**Pull request by dacoex**

### CODE REVIEW

**Comment by spencerahill** [comment]
Thanks for catching this.  I don't know how I managed to mangle that one!

From the badge on the [RTD page](https://readthedocs.org/projects/aospy/), the suggestion is
```
.. image:: https://readthedocs.org/projects/aospy/badge/?version=latest
:target: http://aospy.readthedocs.io/en/latest/?badge=latest
:alt: Documentation Status
```

Would you mind replacing what we have with that instead of what's in your prior commit?  I'll merge then.  The failed test in the py2.7min Travis environment appears to be unrelated.  

You'll be the first to have a merged PR besides myself and @spencerkclark !

**Comment by spencerahill** [comment]
@dacoex I would also like to credit you in our docs for this contribution.  So please also add a note to our [What's New](https://github.com/spencerahill/aospy/blob/develop/docs/whats-new.rst) page under the v0.2.1 section, under a new subheading titled "Documentation".  Thanks!

**Comment by dacoex** [comment]
done

File: README.rst [hunk 1]
```diff
@@ -2,7 +2,7 @@ aospy: automated climate data analysis and management
 =====================================================
 
 .. image:: https://readthedocs.org/projects/pip/badge/?version=stable
-   :target: http://pip.pypa.io/en/stable/?badge=stable
+   :target: http://aospy.readthedocs.io/en/latest/
    :alt: ReadTheDocs Status
```
Comment (spencerahill) [inline README.rst]: Again, please replace these lines with 

```
.. image:: https://readthedocs.org/projects/aospy/badge/?version=latest
:target: http://aospy.readthedocs.io/en/latest/?badge=latest
:alt: Documentation Status
```

File: docs/whats-new.rst [hunk 2]
```diff
@@ -8,6 +8,11 @@ What's New
 v0.2.1 (unreleased)
 -------------------
 
+Documentation
+~~~~~~~~~~~~~
+
+- corrected link to doc badge via :pull:`213`
```
Comment (spencerahill) [inline docs/whats-new.rst]: How about "Corrected link to documentation badge on repository main page (:pull:`213`).  By `DaCoEx <https://github.com/dacoex>`_."

**Comment by spencerahill** [comment]
@dacoex two more comments, then we're ready to go.  Thanks.

**Comment by spencerahill** [comment]
Thanks again @dacoex !

**Pull request merged**